pkg/components/exporter/exporter_test.go

```diff
@@ -0,0 +1,75 @@
+package exporter
+
+import (
+	"testing"
+
+	ko "github.com/sustainable.computing.io/kepler-operator/pkg/api/v1alpha1"
```

Comment:
```suggestion
	"github.com/sustainable.computing.io/kepler-operator/pkg/api/v1alpha1"
```
To match the convention 👼

Reply:
ok